app/uk/gov/hmrc/icedsubscriptionfrontend/views/LandingPage.scala.html

```diff
@@ -46,31 +46,43 @@
 
     @hmrcPageHeading(PageHeading(text = messages("landing.heading")))
 
-    @warningText("landing.fromJanuary", Some("landing.tradersWhoSubmit"))
+    @warningText("landing.fromJanuaryWarning")
 
     @paragraph("landing.coms")
 
     @paragraph("landing.incorporates")
-    @bulletList(Seq(Html(messages("landing.incorporates.1")), Html(messages("landing.incorporates.2"))))
-
-    @warningText("landing.csp", Some("landing.ifYouAreACsp"))
-
-    @heading2("landing.beforeYouStart.heading")
-
-    @paragraph("landing.youNeed")
-    @bulletList(Seq(ggwLink, getEoriLink))
+    @bulletList(Seq(
+        Html(messages("landing.incorporates.1")),
+        Html(messages("landing.incorporates.2"))
+    ))
 
-    @paragraph("landing.takesMinutes")
+    <div id="csps">
+        @heading2("landing.csp.heading")
+        @paragraph("landing.ifYouAreACsp")
+    </div>
+
+    <div id="requirements">
+        @heading2("landing.beforeYouStart.heading")
+
+        @paragraph("landing.youNeed")
+        @bulletList(Seq(
+            ggwLink,
+            getEoriLink,
```

Comment:
these links need to open in a new Tab this can be done by adding `target="_blank"` to the html for the link. Maybe add a Boolean to the component to open in a new tab or not?